meson.build

```diff
@@ -11,7 +11,7 @@ project('libmsg',
 	]
 )
 
-incdir = include_directories('include')
+subdir('include')
```

Comment:
I would like to avoid adding subdirs besides test. It makes the build process more annoying to debug.